pyuvsim/profiling.py

```diff
@@ -45,7 +45,10 @@ def set_profiler(func_list=default_profile_funcs, rank=0, outfile_name='time_pro
         outfile_name: Filename for printing profiling results.
         dump_raw: Write out a pickled LineStats object to <outfile_name>.lprof (Default False)
     """
-    start_mpi()
+    if mpi is None:
+        raise ImportError("You need mpi4py to use the uvsim module. Install pyuvsim[sim]")
```

Comment:
This may not be transparent to users who aren't used to the extras_require syntax. What about:
> You need mpi4py to use the uvsim module. Install it by running `pip install pyuvsim[sim]` or `pip install pyuvsim[all]` if you also want h5py and line_profiler installed.

The other places with similar language should also be updated.